.github/workflows/pr-review-check.yml

```diff
@@ -0,0 +1,37 @@
+name: "PR Review check"
+
+env:
+  TRIGGER_LABELS: '["CI", "preview", "skip-tests"]'
+  DEPENDABOT_CREATOR: 'dependabot[bot]'
+  MILESTONE_NAME: 'Validating'
+
+on:
+  pull_request_review:
+    types:
+      - submitted
+  pull_request:
+    branches:
+      - develop
+    types:
+      - labeled
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger validate on approval
+        if: github.event_name == 'pull_request_review'
+        uses: Alfresco/alfresco-build-tools/.github/actions/github-trigger-approved-pr@fc16d6c2f032a24825ee40131640bc5b4b6736d8 # temp version for testing
+        with:
+          creator: ${{ env.DEPENDABOT_CREATOR }}
+          github-token: ${{ secrets.BOT_GITHUB_TOKEN }}
+          milestone-on-approval: ${{ env.MILESTONE_NAME }}
+          auto-merge-on-approval: true
+
+      - name: Trigger validation on labeling
+        if: github.event_name == 'pull_request'
```

Comment:
Is it expected for this if condition to be so generic vs the step name to mention only 'labeling'?

Reply:
Good point, made the check more specific to "labeled" action :+1: